### DISCUSSION TITLE
[Suggestion] /cmi equip hat/chest/leggings/boots

### DISCUSSION THREAD

**Discussion by ThePyromaniac** [Zrips/CMI#1362] [Feature Requests]
The /cmi hat works in the way I want it to, equipping the item on your head regardless of the item, however I'd like it for all armor slots. (Say for RPG and 3D modeling purposes.)

**Comment by mrfloris** [comment]
So you mean like /mhand, /ohand, /chest /pants /boots ? or something along those lines?

**Comment by ThePyromaniac** [comment]
Yup, something like that.
